airmozilla/starred/tests/test_starred.py

```diff
@@ -0,0 +1,198 @@
+
+from airmozilla.main.models import Event
+from airmozilla.starred.models import (
+    StarredEvents,
+)
+
+
+class TestStarredEvent(TestCase):
+    fixtures = ['airmozilla/manage/tests/main_testdata.json']
+# test that a starred event can be synced
+# test that an unstarred event can be synced
+# test that unchanged events remain that way
+# test that unknown event id does the right thing (whatever that is)
+# test that the proper list is returned from a get
+# test that the proper list of starred events is displayed
+# add star, sync 
+# remove star, add different star, sync
+
+# test what happens when a user isn't logged in
+
+
+    def setUp(self):
+        super(TestStarredEvent, self).setUp()
+
+        # create the url
+        self.url = reverse('starred:sync_starred_events')
+        # so, let's sign in
+        User.objects.create_user('lisa', password='secret')
+        assert self.client.login(username='lisa', password='secret')
+
+        # instantiate test event
+        event = Event.objects.get(title='Test event')
+        # create more events 
+        Event.objects.create(
+            title='A Good Event',
+            slug='event1',
+            description=event.description,
+            start_time=event.start_time,
+            privacy=Event.PRIVACY_PUBLIC,
+            placeholder_img=event.placeholder_img,
+            location=event.location,
+        )
+        Event.objects.create(
+            title='A Great Event',
+            slug='event2',
+            description=event.description,
+            start_time=event.start_time,
+            privacy=Event.PRIVACY_PUBLIC,
+            placeholder_img=event.placeholder_img,
+            location=event.location,
+        )
+        Event.objects.create(
+            title='The Best Event',
+            slug='event3',
+            description=event.description,
+            start_time=event.start_time,
+            privacy=Event.PRIVACY_PUBLIC,
+            placeholder_img=event.placeholder_img,
+            location=event.location,
+        )
+        # get events from the database
+        self.events = Event.objects.all()
+
+    def test_sync_starred_events(self):
+
+        url = self.url
+        events = self.events
+
+        # pass url to the browser
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+
+        # get the empty list of event ids in json format 
+        structure = json.loads(response.content)
+        eq_(structure, {"ids": []})
+
+        # add an event id to the list
+        structure['ids'].append(events[0].id)
+        # send synced list to browser
+        response = self.client.post(
+            url,
+            content_type='application/json',
+            data=json.dumps(structure)
+        )
+        # get the list and verify it was updated
+        structure = json.loads(response.content)
+        eq_(structure, {"ids": [events[0].id]})
+
+    def test_remove_starred_event(self):
+
```

Comment:
This test is called "test_remove_something_something" so then it should ideally be about testing only the remove part. 
One way of doing that is to shortcut to the necessary condition. E.g.:
```python
def test_removed_starred_event(self):
    event1_id = self.events[1].id
    StarredEvent.objects.create(user=self.user, event=self.events[1])
    StarredEvent.objects.create(user=self.user, event=self.events[2])
    response = self.client.post(url, {'ids': [self.events[1].id]})
    assert something on the response
    ok_(not StarredEvent.objects.filter(id=event1_id))
    
```

Do you see? This way, it's only testing one thing; the delete view.